src/index.ts

```diff
@@ -35,48 +35,49 @@ const options: ts.CompilerOptions = ts.readConfigFile(project || 'tsconfig.json'
     .config.compilerOptions || {};
 const host = ts.createCompilerHost(options, true);
 
-const program = ts.createProgram([tsFile], options, host);
+const program = ts.createProgram(tsFiles, options, host);
+let totalNumFailures = 0;
+tsFiles.forEach((tsFile: string) => {
+    const source = program.getSourceFile(tsFile);
+    if (!source) {
+      console.error(`could not load content of ${tsFile}`);
+      process.exit(1);
+    }
+    const scanner = ts.createScanner(
+        ts.ScriptTarget.ES5, false, ts.LanguageVariant.Standard, source.getFullText());
 
-const source = program.getSourceFile(tsFile);
-if (!source) {
-  console.error(`could not load content of ${tsFile}`);
-  process.exit(1);
-}
-const scanner = ts.createScanner(
-    ts.ScriptTarget.ES5, false, ts.LanguageVariant.Standard, source.getFullText());
+    const checker = program.getTypeChecker();
```

Comment:
I just found out that I can't even re-use the program for all the files. working on it, but for now I've committed a very dumb solution.